packages/bot/package.json

```diff
@@ -9,7 +9,8 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
-      "tools.js"
+      "tools.js",
+      "Misc.js"
```

Comment:
Can we change this to `misc.js`?

Reply:
@reza-fs there is a problem with my test so i have to disable it temproray, the file name is `Misc.js` so i'm adding it to jest IgnorePatterns. i think i have to use the filename